src/Entities/MembershipApplication.php

```diff
@@ -11,7 +11,15 @@
 /**
  * @since 2.0
  *
- * @ORM\Table(name="request", indexes={@ORM\Index(name="idx_donation_id", columns={"donation_id"})})
+ * @ORM\Table(
+ *     name="request",
+ *     indexes={
+ *          @ORM\Index(name="idx_donation_id", columns={"donation_id"}),
```

Comment:
This was used as a foreign key in the old application. We dropped it while developing the new shiny one.